integrated_channels/blackboard/exporters/learner_data.py

```diff
@@ -50,9 +50,14 @@ def get_learner_data_records(
             'blackboard',
             'BlackboardLearnerDataTransmissionAudit'
         )
-
-        return [
-            BlackboardLearnerDataTransmissionAudit(
+        course_id = get_course_id_for_enrollment(enterprise_enrollment)
```

Comment:
Have we confirmed if this reutrns a course key or course run key?

Reply:
The course run keys have already been removed. The existing code only returns course key.